src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateCoordinator.swift

```diff
@@ -108,7 +108,13 @@ final class HealthCertificateCoordinator {
 	private var validationCoordinator: HealthCertificateValidationCoordinator?
 	
 	private var infoScreenShown: Bool {
-		get { store.healthCertificateInfoScreenShown }
+		get {
```

Comment:
what's the usage of this change?

Reply:
Good question! 
The usage is, that the info screen has texts like "Add certificates" and that would be not a use case in hibernation, because we only show certificates. Thatswhy I change the getter for hibernation and skip the info screen at this point.

Reply:
I guess that's a first (and maybe the only) move reg. https://github.com/corona-warn-app/cwa-documentation/issues/950?